### PR TITLE
anki: add page

### PR DESCRIPTION
**Pull request by dylancdavis**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages/linux/anki.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# anki
+
+> Powerful, intelligent flashcard program.
+> More information: <https://apps.ankiweb.net/>.
```
Comment (blueskyson) [inline pages/linux/anki.md:4]:
```suggestion
> More information: <https://apps.ankiweb.net>.
```

File: pages/linux/anki.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- Launch anki:
+
+`anki`
+
+- Launch anki with a specific profile:
+
+`anki -p {{profile name}}`
```
Comment (blueskyson) [inline pages/linux/anki.md:12]:
```suggestion
`anki -p {{profile_name}}`
```

**Review by blueskyson — COMMENTED** [review]
This command has Windows client. Please move it to `pages/common/anki.md`.

File: pages/common/anki.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# anki
+
+> Powerful, intelligent flashcard program.
+> More information: <https://apps.ankiweb.net>.
+
+- Launch anki:
```
Comment (kbdharun) [inline pages/common/anki.md:6]:
```suggestion
- Launch `anki`:
```

File: pages/common/anki.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+`anki`
+
+- Launch anki with a specific profile:
```
Comment (kbdharun) [inline pages/common/anki.md:10]:
```suggestion
- Launch `anki` with a specific profile:
```

File: pages/common/anki.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+`anki -p {{profile_name}}`
+
+- Launch anki in a specific langauge:
```
Comment (kbdharun) [inline pages/common/anki.md:14]:
```suggestion
- Launch `anki` in a specific language:
```

File: pages/common/anki.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+`anki -l {{language}}`
+
+- Launch anki from a specifc directory instead of the default (~/Anki):
```
Comment (kbdharun) [inline pages/common/anki.md:18]:
```suggestion
- Launch `anki` from a specific directory instead of the default (`~/Anki`):
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Review by blueskyson — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**